parser-typechecker/tests/Unison/Test/Util/Bytes.hs

```diff
@@ -17,6 +17,17 @@ test = scope "util.bytes" . tests $ [
     ==
     Bytes.fromWord8s [0,1,2,3,4,5,6,7,8,9],
 
+  scope "at" $ do
+    expect' (Bytes.at 0 (Bytes.fromWord8s [77,13,12]) == Just 77)
+    expect' (Bytes.at 0 (Bytes.fromWord8s []) == Nothing)
+    ok,
+
+  scope "at.cornerCases" $ do
+    let b = Bytes.drop 3 $ Bytes.fromWord8s [77,13,12] <> Bytes.fromWord8s [9,10,11]
+    expect' (Bytes.at 0 b == Just 9)
+    expect' (Bytes.at 0 (mempty <> Bytes.fromWord8s [1,2,3]) == Just 1)
+    ok,
```

Comment:
Added a few more unit tests to the `Bytes` type to figure out whether that was the problem here. These tests worked fine even before the fix.